packages/env-tests/src/monorepoRun.ts

```diff
@@ -21,10 +20,32 @@ function runTests() {
   const mnemonic = process.env.MNEMONIC!
   const reserveSpenderMultiSigAddress = process.env.RESERVE_SPENDER_MULTISIG_ADDRESS
 
+  const defaultTokensToTest = ['cUSD']
```

Comment:
If this is a constant, you should extract it to the top-level and caps it

Reply:
makes sense, updated!